lib/src/main/java/vmath/Mat2.java

```diff
@@ -22,6 +22,15 @@ public Mat2(float m00, float m01,
         };
     }
 
+    public Mat2 mul(Mat2 other){
+        float m00, m01, m10, m11;
+        m00 = (m[0]*other.m[0]) + (m[1]*other.m[2]);
+        m01 = (m[0]*other.m[1]) + (m[1]*other.m[3]);
+        m10 = (m[2]*other.m[0]) + (m[3]*other.m[2]);
+        m11 = (m[2]*other.m[1]) + (m[3]*other.m[3]);
```

Comment:
The parentheses are not necessary here and it would be better if they were removed